maintainers/maintainer-list.nix

```diff
@@ -13533,6 +13533,13 @@
     githubId = 28183516;
     name = "Mateusz Słodkowicz";
   };
+  mateusauler = {
+    email = "mateus@auler.dev";
+    github = "mateusauler";
+    githubId = "24767687";
+    name = "Mateus Auler";
+    keys = [ { fingerprint = "A09D C093 3C37 4BFC 2B5A  269F 80A5 D62F 6EB7 D9F0"; } ];
```

Comment:
Format the file. The `keys` attribute will triger `nixpkgs`' Ci/Cd check. Go into a `nix-shell` from the root of `nixpkgs` and run `nix-fmt maintainers/maintainer-list.nix`.

PS: The commands are from memory. If they don't work look at the docs or one of the Actions where I failed to format things (it described there as well).

> [!Note]
> Don't tryto format everything, it won't finish today XD

Reply:
I got the commands for you from [a failed action](https://github.com/NixOS/nixpkgs/actions/runs/11803569498/job/32881816498):

```
Please go to the Nixpkgs root directory, run `nix-shell`, then:
nixfmt 'maintainers/maintainer-list.nix'
```

Reply:
Forgot about that :P
I tried running nixfmt, but it did nothing.
So I did it manually. Does it look good?